frame/society/src/lib.rs

```diff
@@ -1250,7 +1296,13 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 					<Vouching<T, I>>::remove(&voucher);
 				}
 			}
-			Self::deposit_event(RawEvent::AutoUnbid(popped));
+			Self::deposit_event(Event::AutoUnbid(popped));
+
+			match insert_pos {
+				Some(p) => bids.try_insert(p, bid_to_insert),
+				None => bids.try_push(bid_to_insert),
+			}
+			.expect("b.len() < 1000; qed");
```

Comment:
I think this should be a log message versus an expect.

If above, b.len could be more than 1000, then this assertion cannot be strictly true after only a single pop